extras/minimal-printf/README.rst

```diff
@@ -0,0 +1,51 @@
+======================================
+Minimal printf replacement for Duktape
+======================================
+
+The ``duk_minimal_printf.c`` provides a portable provider for ``printf()``
+with a feature set matching minimally what Duktape needs.  The provider
+compiles to less than 1kB.
+
+Assumptions:
+
+* ``sizeof(void *) <= sizeof(long)``
```

Comment:
I know - this is mostly useful on low memory targets where this is not an issue.